Simplified/NYPLReaderReadiumView.m

```diff
@@ -871,8 +878,30 @@ - (void)readiumPaginationChangedWithDictionary:(NSDictionary *const)dictionary
           spineItemTitle:weakSelf.spineItemDetails[@"tocElementTitle"]];
        }];
 
-       NYPLBookLocation *const location = [[NYPLBookLocation alloc] initWithLocationString:locationJSON renderer:renderer];
-       NSString *const bookID = weakSelf.book.identifier;
+      float chapterProgress = [self progressWithinChapter];
```

Comment:
minor: indentation is off

Reply:
yeah... actually the off part is in the rest of the code, which I almost fixed but decided not to, to avoid conflicts when I merge this code with the newer R2 changes. Since this whole file will be deleted soon once R2 is done, I just left it as is.